src/Analysis/GWASWizard/shared/GWASFormSubmit.jsx

```diff
@@ -163,7 +161,7 @@ const GWASFormSubmit = ({
           className='GWASUI-nameInput'
           onChange={handleGwasNameChange}
           value={gwasName}
-          placeholder='Enter a job name...'
+          placeholder='Enter a study name'
```

Comment:
"Give a name to your study" or "Give a name to your run" is probably more accurate. "Enter a name" feels like it is going to look up this name somewhere.

Reply:
I like "give a name to your study"